nearcore/src/config.rs

```diff
@@ -1077,6 +1096,30 @@ pub fn download_genesis(url: &String, path: &PathBuf) {
             .await
             .expect("Genesis file is bigger than 10GB. Please make the limit higher.");
 
+        std::fs::write(&path, &body).expect("Failed to create / write a genesis file.");
+
+        info!(target: "near", "Saved the genesis file to: {} ...", path.as_path().display());
+    });
+}
+
+pub fn download_config(url: &String, path: &PathBuf) {
+    info!(target: "near", "Downloading config file from: {} ...", url);
+    let url = url.clone();
+    let path = path.clone();
+
+    actix::System::new().block_on(async move {
+        let client = awc::Client::new();
+        let mut response =
+            client.get(url).send().await.expect("Unable to download the config file");
+
+        // IMPORTANT: limit specifies the maximum size of the genesis
```

Comment:
Would this work for testnet genesis?

Reply:
Yes, not sure why wouldn't it?

Reply:
Verified that it works on testnet: https://github.com/near/nearcore/pull/4427#issuecomment-871465135